src/lib/datepicker/datepicker-input.ts

```diff
@@ -224,7 +224,7 @@ export class MdDatepickerInput<D> implements AfterContentInit, ControlValueAcces
   ngAfterContentInit() {
     if (this._datepicker) {
       this._datepickerSubscription =
-          this._datepicker.selectedChanged.subscribe((selected: D) => {
+        this._datepicker.selectedChanged.subscribe((selected: D) => {
```

Comment:
Nit: this should stay the same too. They indent 2 for function blocks and 4 for line breaks